lighthouse-core/audits/dobetterweb/no-datenow.js

```diff
@@ -57,7 +57,7 @@ class NoDateNowAudit extends Audit {
     const pageHost = url.parse(artifacts.URL.finalUrl).host;
     // Filter usage from other hosts.
     const results = artifacts.DateNowUse.usage.filter(err => {
-      return url.parse(err.url).host === pageHost;
+      return url.parse(err.url || '').host === pageHost;
```

Comment:
@ebidel 
The displayed that `url` was `undefined` like below image.  What do you think about display? I think that `-` is more good than `undefined`.

![image](https://cloud.githubusercontent.com/assets/22300/19939927/41bd3132-a16e-11e6-91cb-38c54fd6a49e.png)